cpp/unreal_plugins/SimulationController/Source/SimulationController/RpcServer.h

```diff
@@ -156,14 +161,258 @@ struct FunctionWrapper<TReturn (*)(TArgs...)>
 
 } // namespace detail
 
-template <typename TFunctor>
-inline void RpcServer::bindAsync(const std::string& name, TFunctor&& functor)
+
```

Comment:
Do we still need the `wrapSync` function on Line 141?

Reply:
No we do not. I didn't want to put more effort than necessary into this file though because you're in the middle of refactoring it. Maybe I shouldn't check in any changes to this file at all, because it'll cause downstream problems when you are merging your latest batch of changes.

Reply:
I agree that not modifying this file would be better as it'll help avoid unnecessary merging issues later on as you mentioned.

Reply:
Fixed. I removed all the changes from `RpcServer.h` except I needed to keep my adjustments to the `#includes` so the code would compile.